resources/views/community/components/recent-awards/meta-type-filter.blade.php

```diff
@@ -0,0 +1,15 @@
+@props([
+    'selectedAwardType' => null,
+])
+
+<div class="grid gap-y-1 sm:pl-[30px] sm:pr-8">
+    <label class="text-xs font-bold" for="filter-by-award-kind">Award kind</label>
+
+    <select id="filter-by-award-kind" class="w-full" onchange="handleAwardTypeChanged(event)">
```

Comment:
Changing the Award filter should likely reset the offset since it will not be uncommon if you move to a later page and then filter down to just a specific award type that there will be nothing appearing in the list without moving going back manually

Reply:
I think this may be confusing for the user experience though. Since we can't set multiple filters simultaneously, I find myself changing the date first and then setting the award filter I want (largely because date is the first filter in the filter list).

Reply:
Sorry to clarify a bit I'm just suggesting to reset the offset (next/prev 25). Ex 
I picked some date and see various awards (Mastered and Completed). I click next 25 and see more awards of each type for that day. Then I change the filter to Completed awards which I saw some on the first and second group of 25 users for that date and now I see none because I am still on the second page (1 offset over) so I'm confused because I know I saw some Completed awards but now none are showing.

Reply:
Ah, sorry, I misunderstood. You're right, I will make this change.

Reply:
Should be all set in latest.

Reply:
minor: the page offset doesn't reset when switching between "all" and "followed"

Reply:
I have no way to test a fix for this and have any confidence in my change.

It will have to wait until new DB dumps are produced.

Reply:
Resolved in https://github.com/RetroAchievements/RAWeb/pull/1782/commits/ff7198c90d19b0e863395e20382f3f9c76fce4b3.